### PR TITLE
SAGE-186: Sage 3.0 update Alert small variant

### PR DESCRIPTION
**Pull request by monicawheeler**
Resolves Jira issue [SAGE-186](https://kajabi.atlassian.net/browse/SAGE-186)

## Description

Align Alerts with the **[design specs](https://www.figma.com/file/9Km09NjlZHYWsMP7EGT8tI/Sage-3-for-Admin?node-id=13280%3A22961)**

## Screenshots
|  Before  |  After  |
|--------|--------|
| before-alerts-sm | after-alerts-sm |

## Testing in `sage-lib`
1. Navigate to http://localhost:4000/pages/component/alert
2. Verify standard size and small alerts align with **[design specs](https://www.figma.com/file/9Km09NjlZHYWsMP7EGT8tI/Sage-3-for-Admin?node-id=13280%3A22961)**

## Testing in `kajabi-products`
1. (**MEDIUM**) Review app instances of alerts to confirm no regressions. Some examples of areas for QE to verify
   - [ ] Fields managed by Calendly integration alert
   - [ ] Cannot merge contacts alert
   - [ ] PayPal connection in progress alert
   - [ ] Upgrade account alert for custom email domains

## Related
[SAGE-186](https://kajabi.atlassian.net/browse/SAGE-186)

### CODE REVIEW

File: packages/sage-assets/lib/stylesheets/components/_alert.scss [hunk 1]
```diff
@@ -18,18 +18,38 @@ $-alert-colors: (
...
   display: grid;
-  grid-auto-columns: auto;
-  grid-auto-flow: column;
-  grid-template-columns: auto 1fr;
   gap: sage-spacing(sm);
   position: relative;
-  margin-bottom: sage-spacing(md);
-  padding: sage-spacing(sm);
-  border-radius: sage-border(radius);
+  padding: sage-spacing();
```
Comment (anechol) [inline packages/sage-assets/lib/stylesheets/components/_alert.scss:23]: Looks like our padding for the standard size is `16px`. Can we update this?

File: packages/sage-assets/lib/stylesheets/components/_alert.scss [hunk 2]
```diff
@@ -73,7 +97,7 @@ $-alert-colors: (
 }
 
 .sage-alert__title {
-  @extend %t-sage-heading-6;
+  @extend %t-sage-heading-5;
```
Comment (anechol) [inline packages/sage-assets/lib/stylesheets/components/_alert.scss:100]: Could we update this to `14px` as well?

**Review by philschanely — APPROVED** [review]
🙌

**Review by anechol — CHANGES_REQUESTED** [review]
Just a couple things!

**Review by QuintonJason — APPROVED** [review]

**Review by anechol — CHANGES_REQUESTED** [review]
Changes are good, but I've just noticed the border radius is `6px` in Figma. That should be the last change.

**Comment by monicawheeler** [comment]
@anechol - great catches on this! Thank you 🙌

**Review by anechol — APPROVED** [review]
Nice, LGTM

**Pull request merged**